docs/data/install-bestman-xrootd.md

```diff
@@ -0,0 +1,264 @@
+
+1. On the xrootd redirector node, execute the following command:
+
+        :::console
+        root@host # xrdsssadmin -k %RED%<my_key_name>%ENDCOLOR% -u anybody -g usrgroup add %RED%<keyfile>%ENDCOLOR%
+
+        eg:
+
+        root@host # xrdsssadmin -k top_secret -u anybody -g usrgroup add /etc/xrootd/xrootd.key
+
+2. Set ownership
+
+        :::console
+        root@host # chown xrootd.xrootd /etc/xrootd/xrootd.key
+
+3. On the node where xrootdfs is installed modify **`/etc/fstab`** add security information:
+
+        :::console
+        root@host # xrootdfs %RED%/mnt/xrootd %ENDCOLOR" fuse rdr=xroot://%RED%redirector1.domain.com%ENDCOLOR%:1094/%RED%/path/redirector1%ENDCOLOR%,uid=xrootd,sss=%RED%keyfile%ENDCOLOR% 0 0
+
+4. On all xrootd data servers and redirector node, modify xrootd configuration (**`/etc/xrootd/xrootd-clustered.cfg`**) by adding the following segment: 
+
+        :::file
+        # ENABLE_SECURITY_BEGIN 
+           xrootd.seclib /usr/lib64/libXrdSec.so 
+           #the line below should be before "sec.protocol ... unix"
+           %RED%sec.protocol /usr/lib64 sss -s keyfile %ENDCOLOR%
+           sec.protocol /usr/lib64 unix 
+           # this specify that we use the 'unix' authentication module, additional one can be specified. 
+           # this is the authorization file
+           acc.authdb /etc/xrootd/auth_file
+           ofs.authorize 
+           # ENABLE_SECURITY_END
+
+5. On all xrootd data server nodes, edit /etc/xrootd/auth\_file to add authorized users of the form `u %RED%username%ENDCOLOR% %RED%/directoryname%ENDCOLOR% lr` where "lr" is the permission set.
+
+6. Copy %RED%keyfile<span class="twiki-macro ENDCOLOR"></span> from redirector node to every data server node and the xrootdfs node. Make sure that this file is owned by the `xrootd` user.
+
+7. Restart xrootd cluster by following [these instructions](https://twiki.grid.iu.edu/bin/view/SoftwareTeam/HowToInstallXrootd)
+8. On xroodfs node execute mount:
+
+        :::console
+        root@host # mount %RED%/mnt/xrootd%ENDCOLOR%
+
+9. Verify that you can access the mount point (df,ls) and can not write into unauthorized path, e.g:
+
+        :::console
+        root@host # cp /bin/sh /mnt/xrootd/tlevshin/test1 cp: 
+        cannot create regular file \`/mnt/xrootd/tlevshin/test1': Permission denied
+
+    Login as yourself and try:
+ 
+        :::console
+        root@host # su - tlevshin 
+        user@host $ cp /bin/sh /mnt/xrootd/tlevshin/test1
+
+Edit Bestman Settings
+---------------------
+
+See [Edit Bestman Settings](../data/bestman-install/#edit-bestman-settings)
+
+Modify `/etc/sudoers`
+---------------------
+
+See [Bestman Instructions](../data/bestman-install/#modify-etcsudoers)
+
+### (Optional) Copying certificates to a bestman location
+
+See [Bestman Instructions](../data/bestman-install/#copying-certificates-to-an-alternate-location)
+
+
+Configure Xrootd Gratia Probes
```

Comment:
I don't really think any of this works anymore.  I would remove the whole section unless we see some motivation to add it back.